mkosi/__init__.py

```diff
@@ -1450,6 +1461,10 @@ def check_inputs(config: MkosiConfig) -> None:
             if not p.is_file():
                 die(f"Initrd {p} is not a file")
 
+    if (want_uki(config) and
+        not (shutil.which("ukify") or Path("/usr/lib/systemd/ukify").exists())):
+        die(f"'ukify' not available")
```

Comment:
Don't need the f string here, it can be a normal one

Reply:
Also I'd say "ukify not found"

Reply:
And add a hint to tell users to set Bootable=no if they don't want a bootable image